lfs/util.go

```diff
@@ -115,6 +115,11 @@ func FilenamePassesIncludeExcludeFilter(filename string, includePaths, excludePa
 	if len(includePaths) > 0 {
 		matched := false
 		for _, inc := range includePaths {
+			// Special case local dir, matches all (inc subpaths)
+			if inc == "." || inc == "./" || inc == ".\\" {
+				matched = true
+				break
+			}
```

Comment:
I was trying to avoid polluting the namespace unnecessarily but I guess it'll get called a lot so worth it.

BTW using a map[]struct{} instead of a map[]bool saves 1 byte per entry since the empty struct takes 0 bytes. \o/ :D